kolibri/plugins/management/assets/src/vue/class-edit-page/index.vue

```diff
@@ -23,18 +23,17 @@
     </div>
 
     <div class="toolbar">
-      <div class="searchbar" role="search">
+      <div class="searchbar" :class="{'searchbar-mobile': elSize.width < 600}" role="search">
```

Comment:
* I'd love to have a relatively small set of constants to use from for element query if we know what these constants are.
* the side-menu causes a situation where the window size not changed but the element size gets changed a lot. That's why breakpoints not suitable here.
* you have the designs first, later comes the implementation detail. It's like the present won't make the past easier or harder.